Cargo.toml

```diff
@@ -1,3 +1,7 @@
+[workspace]
+members = [".", "cargo-insta", "cargo-insta/integration-tests"]
+default-members = [".", "cargo-insta"]
```

Comment:
This `default-members` setting is what enables `cargo run` to work in the repository root.